src/main/java/org/ow2/proactive/catalog/service/CatalogObjectService.java

```diff
@@ -432,7 +433,7 @@ public CatalogObjectMetadata restore(String bucketName, String name, Long commit
     @VisibleForTesting
     protected CatalogObjectRevisionEntity getCatalogObjectRevisionEntityByCommitTime(String bucketName, String name,
             long commitTime) {
-        CatalogObjectRevisionEntity revisionEntity = catalogObjectRevisionRepository.findCatalogObjectRevisionByCommitTime(bucketName,
+        CatalogObjectRevisionEntity revisionEntity = catalogObjectRevisionRepository.findCatalogObjectRevisionByCommitTime(Arrays.asList(bucketName),
```

Comment:
The called method should not do any changes to your structure? So `Collections.singletonList` might be more appropriate here